src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadCreate.js

```diff
@@ -36,16 +43,21 @@ const AnnotationThreadCreate = ({
     target,
 }: Props) => {
     const [isPending, setIsPending] = React.useState(false);
+    const events = useAnnotatorEvents({ eventEmitter });
```

Comment:
Minor, but you could potentially do destructuring assignment of `emitAddAnnotationStartEvent` and `emitAddAnnotationEndEvent` here.

Reply:
I agree with @kkuliczkowski-box, we should destructure the handlers here.